### DISCUSSION TITLE
Misleading field names at: Backend / Configuration / SEO

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3030] [Ideas]
There are a number of different **meta tags**:

- description
- keywords
- robots
- property="og:title"
- name="twitter:card"
- and a huge lot more...

Accordingly, "meta tags" is **not sufficient** as a field name for Backend, Configuration, SEO.

**meta robots** would be better!

Especially because there will be **more meta fields in the future**.

Regards, Christian

**Comment by thorsten** [comment]
The naming will be better with 3.2.8.
